aws/cloudformation/ami.yml.erb

```diff
@@ -4,58 +4,51 @@ unless dry_run
   update_bootstrap_script.call
 end
 
-# Nonce value for AMI resources. Change this value to generate a new base AMI on the next CloudFormation update.
-ami = 1
+commit = ENV['COMMIT'] || `git ls-remote origin #{branch}`.split.first
+ami = commit[0..4]
 -%>
 ---
 AWSTemplateFormatVersion: 2010-09-09
 Description: AWS CloudFormation Template for Code.org application
 Resources:
-  VPC: <%= lambda_fn.call 'LookupStackOutputs', StackName: 'VPC' %>
-  IAM: <%= lambda_fn.call 'LookupStackOutputs', StackName: 'IAM' %>
```

Comment:
All `LookupStackOutputs` references are removed, and `Fn::GetAtt` function-references using these resources are replaced with `!ImportValue` function-references.